Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.45372&attempt_id=c16f52f9-5f13-4a72-a034-94464f938694)

Attempted: facette

<details><summary>Partial log (click to expand)</summary><p>

```
npm ERR! network
npm ERR! network If you are behind a proxy, please make sure that the
npm ERR! network 'proxy' config is set properly.  See: 'npm help config'

npm ERR! A complete log of this run can be found in:
npm ERR!     /build/npm-1000/.npm/_logs/2018-08-20T09_49_17_946Z-debug.log
result: fail
make: *** [Makefile:153: node_modules] Error 1
builder for '/nix/store/xr50yxa2ca2yfcasdzgqwlr4z28d9m1w-facette-0.4.1.drv' failed with exit code 2
error: build of '/nix/store/xr50yxa2ca2yfcasdzgqwlr4z28d9m1w-facette-0.4.1.drv' failed
```
</p></details>

--- a/pkgs/servers/monitoring/facette/default.nix
+++ b/pkgs/servers/monitoring/facette/default.nix
@@ -26,5 +26,6 @@ stdenv.mkDerivation rec {
     homepage = https://facette.io/;
     license = licenses.bsd3;
     maintainers = with maintainers; [ fgaz ];
+    broken = true; # not really broken, it just requires an internet connection to build. see #45382
   };
 }

